openshift/ci/dockerfiles/ansible-e2e.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM basebuilder AS builder
+
+RUN mkdir -p /ansible
+COPY openshift/ci/testdata/memcached-operator /ansible/memcached-operator/
```

Comment:
Do we need builder? we can directly copy `requirements.yml`, `watches.yaml`, `/roles/` and `/playbooks/`.

Reply:
Thanks for the suggestion; yeah, since we are not executing any command other than coping the test data, we can skip the multi-stage build. Updated in the latest commit